### ISSUE TITLE
[Feature Request] Plan.retrieve implementation that supports expanding fields.

### ISSUE THREAD

**Issue by majelbstoat** [stripe/stripe-java#559]
The Stripe docs demonstrate how to do expansion:

```java
Stripe.apiKey = "{key}";

List<String> expandList = new LinkedList<String>();
expandList.add("customer");

Map<String, Object> params = new HashMap<String, Object>();
params.put("expand", expandList);

Charge.retrieve("{charge_id}", params, null);
```

And Plans have `product` as an expandable field. But, there is no method signature for retrieving a Plan with params like there is for Charge.

At the moment, I'm working around it by subclassing, but it would be nice if I didn't have to:

```java
import com.stripe.exception.*;
import com.stripe.model.Plan;
import com.stripe.net.RequestOptions;

import java.util.Map;

public class StripePlan extends Plan {
    /**
     * Retrieve a plan.
     */
    public static Plan retrieve(String id, Map<String, Object> params, RequestOptions options)
            throws AuthenticationException, InvalidRequestException,
            APIConnectionException, CardException, APIException {
        return request(RequestMethod.GET, instanceURL(Plan.class, id), params, Plan.class, options);
    }
}
```

**Comment by remi-stripe** [comment]
@majelbstoat Thanks for the detailed report! We should definitely fix this! Would you be up for opening a PR since you have the code ready? I can take otherwise!

**Comment by majelbstoat** [comment]
Sure thing :)

**Comment by ob-stripe** [comment]
Hi @majelbstoat! I've just released a new version of the library (6.2.0). All `retrieve()` methods now have a signature that accepts a parameters map.

Thanks for bringing this to our attention!

**Comment by majelbstoat** [comment]
Awesome, thanks for the fast turnaround :)
